src/nile/cli.py

```diff
@@ -144,7 +144,8 @@ def test(contracts):
 @cli.command()
 @click.argument("contracts", nargs=-1)
 @click.option("--directory")
-def compile(contracts, directory):
+@click.option("--account_contract", flag_value="--account_contract")
```

Comment:
why is `flag_value="--account_contract"` necessary? instead of e.g. using `is_flag=True`

Reply:
I thought it might look cleaner, so the value that gets passed down to `core/compile` is either `False` or `--account_contract`. On the other hand, it might be easier to read in `core/compile` if the `--account_contract` string is hardcoded right there. Easy fix if preferred, what do you think?

Reply:
`--account_contract` is a CLI flag, while the `core/compile` module is a building block (ideally agnostic of external interfaces such as CLI, scripting API, or whatever). I wouldn't leak a CLI flag to the core module.